core/v0/api/core.yaml

```diff
@@ -1872,14 +1873,14 @@ components:
         tags:
           $ref: '#/components/schemas/Tags'
     Gps:
-      description: Describes a gps coordinate
+      description: Describes a GPS coordinates which is a unique identifier of a precise geographic location on the earth.<br>
       type: string
       pattern: '^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?),\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$'
     Image:
       description: 'Image of an object. <br/><br/> A url based image will look like <br/><br/>```uri:http://path/to/image``` <br/><br/> An image can also be sent as a data string. For example : <br/><br/> ```data:js87y34ilhriuho84r3i4```'
```

Comment:
As a best practice, example URLs should always show `https`. The same image URL may be corrected to say `https` instead of `http`.